src/sdkg.rs

```diff
@@ -690,7 +690,7 @@ mod tests {
 
     #[test]
     fn test_threshold() {
-        for nodes_num in 2..10 {
+        for nodes_num in 2..7 {
```

Comment:
Just to make sure we include the N=7 case that we'll be using for elders.
```suggestion
        for nodes_num in 2..=7 {
```